src/test/style.ts

```diff
@@ -9,6 +9,10 @@ var patch = init([
   styleModule
 ]);
 
+const featureDiscoveryElm = document.createElement('div')
+featureDiscoveryElm.style.setProperty('--foo', 'foo')
+const hasCssVariables = featureDiscoveryElm.style.getPropertyValue('--foo') === 'foo'
```

Comment:
IE11 does not have CSS variables. Simply skip those tests in it. I'm OK with that.